sdv/sequential/par.py

```diff
@@ -76,13 +76,24 @@ def _get_context_metadata(self):
 
         for column in context_columns:
             context_columns_dict[column] = self.metadata.columns[column]
+            # Context datetime SDTypes for PAR have already been converted to float timestamp
+            if context_columns_dict[column]['sdtype'] == 'datetime':
+                context_columns_dict[column] = {'sdtype': 'numerical'}
```

Comment:
do we know why we transform datetimes ourselves instead of skipping them like we do for every other sdtype? I see datetime is excluded from this list, but I'm not sure why. Can PAR not handle transforming the datetimes itself?
https://github.com/sdv-dev/SDV/blob/075cdffecae62c103ff4f93d32e54f10fb064945/sdv/sequential/par.py#L69

Reply:
The par requires the data datetime data to be `Continuous` or `Timestamp` . We transform the data in order to make sure that it is in continuous / timestamp space, otherwise here we could have a string representing a datetime and that ends up being a category instead of continuous or timestamp value.
As per:
https://github.com/sdv-dev/DeepEcho/blob/main/deepecho/models/par.py#L65-L84